src/library_wasi.js

```diff
@@ -194,6 +194,21 @@ var WasiLibrary = {
     return 0;
   },
 
+#ifdef WASM_BIGINT
```

Comment:
I think instead of ifdefing we can use the macros, see e.g. clock.get
```javascript
  clock_time_get: function(clk_id, {{{ defineI64Param('precision') }}}, ptime) {
    {{{ receiveI64ParamAsI32s('precision') }}}
```